pkg/square/square.go

```diff
@@ -93,7 +93,7 @@ func (s Square) Size() uint64 {
 }
 
 func Size(len int) uint64 {
-	return uint64(math.Ceil(math.Sqrt(float64(len))))
+	return shares.RoundUpPowerOfTwo(uint64(math.Ceil(math.Sqrt(float64(len)))))
 }
```

Comment:
with its normal use, we don't need to do this because the len is always a power of two due to square construction, but if we want to use this value outside of that context, we need to add this

this change isn't directly releated to setting the defaults, but I think its needed for implementing some restriction on square size